create_derivatives/fixtures/manifest_generation_bag/3aai9usY3AZzCSFkB3RSQ8/data/MANIFEST/asdfjklmn.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
If I am reading tests right/seeing the right things here, it looks like you test for the existence of this file, but it's already checked into version control so it will always be there? Is there a reason this generated manifest is checked in?

Reply:
Good catch, that was unintentional. I'm removing the fixture after it has been created in the tests so those tests remain meaningful.